features/command_line/bisect.feature

```diff
@@ -28,8 +28,8 @@ Feature: Bisect
         end
       end
       """
-    And files "spec/calculator_2_spec.rb" through "spec/calculator_9_spec.rb" with an unrelated passing spec in each file
-    And a file named "spec/calculator_10_spec.rb" with:
+    And files "spec/calculator_2_spec.rb" through "spec/calculator_19_spec.rb" with an unrelated passing spec in each file
```

Comment:
I increased the number of files to 20, because with 10 the bisect would complete before abort with Ctr-C.